Submission of Lab 02 by BT23F05F003

akshay-ambulge opened the pull request:
Demonstrated merge sort recursively.

akshay-ambulge committed:
Complete lab 2

Made change in a task program

akshay-ambulge commented:
@bot-s-m-quadri status

bot-s-m-quadri commented:
## 📌 PRN: BT23F05F003

We've successfully received your submission for **Lab 02**.

**Your PRN**: `BT23F05F003`  
**Lab Number**: 02  
**Base Branch**: `lab-daa-02`

### All your submissions so far:
- #102
- #75
- #57
- #30

Please ensure you complete all required files for this lab.  
We will review your submissions **within at most 3 weeks**, i.e., by **11 September 2025**.

If you have doubts or questions, comment here, and we’ll get back to you  
once you’ve completed your work.

_Thank you for your submission._

akshay-ambulge committed:
Completed lab 2

I've completed lab 2 by solving the 3 tasks which required merge sort approach in all of them.

s-m-quadri commented:
👍🏻 Good work.

@bot-s-m-quadri merge it.

bot-s-m-quadri commented:
👍 Got it, I’ll start merging and organizing your submission.
